app/src/main/java/ar/com/wolox/android/cookbook/maps/MapFragment.kt

```diff
@@ -0,0 +1,119 @@
+package ar.com.wolox.android.cookbook.maps
+
+import android.Manifest
+import android.annotation.SuppressLint
+import androidx.core.app.ActivityCompat
+import ar.com.wolox.android.cookbook.R
+import ar.com.wolox.android.cookbook.databinding.FragmentMapBinding
+import ar.com.wolox.wolmo.core.fragment.WolmoFragment
+import ar.com.wolox.wolmo.core.permission.PermissionListener
+import com.google.android.gms.maps.CameraUpdateFactory
+import com.google.android.gms.maps.GoogleMap
+import com.google.android.gms.maps.OnMapReadyCallback
+import com.google.android.gms.maps.SupportMapFragment
+import com.google.android.gms.maps.model.LatLng
+import com.google.maps.android.clustering.ClusterItem
+import com.google.maps.android.clustering.ClusterManager
+
+class MapFragment : WolmoFragment<FragmentMapBinding, MapPresenter>(), MapView,
```

Comment:
MapPresenter is an empty class, so you can do this: `WolmoFragment<FragmentMapBinding, Any>()`